content/init-huaweicloud.md

```diff
@@ -0,0 +1,104 @@
+ 1. On the VPC Dashboard, select **Security group**.
+ 1. Select a region and then click **Create Security Group**.
+ 1. Complete the Create Security Group form with the following information:
+    * **Security group name**: bosh
+    * **Description**: BOSH deployed VMs
+    * **VPC**: Select the "bosh" VPC that you created in [Create a Virtual Private Cloud](#create-vpc).
+ 1. Select the created security group with group name "bosh", in the Actions column click Configure Rules.
+ 1. On the Security Group Rules page, click Add Security Group Rules.
+ 1. Fill out the Edit inbound rules form and click **Save**.
+     !!! note
+        It highly discouraged to run any production environment with <code>0.0.0.0/0</code> source or to make any BOSH management ports publicly accessible.
+     <table border="1" class="nice">
+      <tr>
+        <th>Type</th>
+        <th>Port Range</th>
+        <th>Source</th>
+        <th>Purpose</th>
+      </tr>
+       <tr><td>Custom TCP Rule</td><td>22</td><td>(My IP)</td><td>SSH access from CLI</td></tr>
+      <tr><td>Custom TCP Rule</td><td>6868</td><td>(My IP)</td><td>BOSH Agent access from CLI</td></tr>
+      <tr><td>Custom TCP Rule</td><td>25555</td><td>(My IP)</td><td>BOSH Director access from CLI</td></tr>
+       <tr><td>All TCP</td><td>0 - 65535</td><td>ID of this security group</td><td>Management and data access</td></tr>
+      <tr><td>All UDP</td><td>0 - 65535</td><td>ID of this security group</td><td>Management and data access</td></tr>
+    </table>
+ See [Creating a Security Group](https://www.alibabacloud.com/help/doc-detail/25468.htm)
+ See [Add security group rules](https://www.alibabacloud.com/help/doc-detail/25471.htm)
+ ---
+## Step 2: Deploy {: #deploy }
+ 1. Install [CLI v2](cli-v2.md).
+ 1. Use `bosh create-env` command to deploy the Director.
+     ```shell
+    # Create directory to keep state
+    $ mkdir bosh-1 && cd bosh-1
+     # Clone Director templates
+    $ git clone https://github.com/cloudfoundry/bosh-deployment
+     # Fill below variables (replace example values) and deploy the Director
+    $ bosh create-env bosh-deployment/bosh.yml --state=state.json \
+ --vars-store=creds.yml \
+ -o bosh-deployment/huaweicloud/cpi.yml \
+ -v director_name=my-bosh \
+ -v internal_cidr=192.168.0.0/24 \
+ -v internal_gw=192.168.0.1 \
+ -v internal_ip=192.168.0.2 \
+ -v subnet_id=... \
+ -v default_security_groups=[bosh] \
+ -v region=cn-north-1 \
+ -v auth_url=https://iam.cn-north-1.myhwclouds.com/v3 \
+ -v az=cn-north-1a \
+ -v default_key_name=bosh \
+ -v huaweicloud_password=... \
+ -v huaweicloud_username=... \
+ -v huaweicloud_domain=... \
+ -v huaweicloud_project=cn-north-1 \
+ -v private_key=bosh.pem
+
+    ```
+     If running above commands outside of an HuaweiCloud VPC, refer to [Exposing environment on a public IP](init-external-ip.md) for additional CLI flags.
+     See [HuaweiCloud CPI errors](HuaweiCloud-cpi-errors.md) for list of common errors and resolutions.
```

Comment:
The `HuaweiCloud-cpi-errors.md` page seems missing from the PR ?

Reply:
Done Thanks